back/index.js

```diff
@@ -36,7 +36,7 @@ app.route('/register').post(async (request, response) => {
 		const user = { email: request.body.email, password: hashedPassword }
 
 		connection.query(
-			'SELECT email FROM account WHERE email = ?',
+			`SELECT email FROM account WHERE email = ${user.email}`,
```

Comment:
no: this makes you vulnerable to sql injection. try:

    		connection.query(
			'SELECT email FROM account WHERE email = ?',
                        [user.email],
                        ...